### PR TITLE
:ghost: who you gonna call

### PR DESCRIPTION
**Pull request by ajthomascouk**

### CODE REVIEW

**Comment by c2s-dev** [comment]
:rocket: deployment of gp-finder succeeded (http://gp-finder-pr-123.dev.beta.nhschoices.net)

File: app/views/layout.nunjucks [hunk 1]
```diff
@@ -20,7 +20,7 @@
     <meta name="DCSext.ServiceName" content="GP Appointment Finder" />
     {% block meta %}{% endblock %}
 
-    {% set qs = '?v=5' %} <!-- DO NOT USE 6 -->
+    {% set qs = '?v=6' %} <!-- DO NOT USE 6 -->
```
Comment (st3v3nhunt) [inline app/views/layout.nunjucks]: Is there a reason the comment that says `DO NOT USE 6` is being ignored? Please change it to 7. You might as well delete the comment at the time.

Comment (ajthomascouk) [inline app/views/layout.nunjucks]: Didn't even notice that! :see_no_evil:

**Comment by ajthomascouk** [comment]
@st3v3nhunt wrote:
> Please rebase at the same time as updating the version.

Missed this comment. grr

**Comment by c2s-dev** [comment]
:rocket: deployment of gp-finder succeeded (http://gp-finder-pr-123.dev.beta.nhschoices.net)

**Pull request merged**